docs/iris/src/whatsnew/latest.rst

```diff
@@ -176,6 +187,10 @@ Documentation
 * Added a warning to the :func:`iris.analysis.cartography.project` function
   regarding its behaviour on projections with non-rectangular boundaries.
 
+* Added an explanation to the user guide of how ``Units`` are handled during
```

Comment:
It would be useful to have a link to the section of documentation that is added.  I assume it would be to this:

https://github.com/SciTools/iris/blob/master/docs/iris/src/userguide/cube_maths.rst#combining-units

Suggest adding a `.. _cube-maths_units:` anchor and then linking from this whats new entry.